src/core/Akka/Dispatch/AbstractDispatcher.cs

```diff
@@ -120,8 +119,26 @@ internal sealed class ChannelExecutorConfigurator : ExecutorServiceConfigurator
     {
         public ChannelExecutorConfigurator(Config config, IDispatcherPrerequisites prerequisites) : base(config, prerequisites)
         {
-            var cfg = config.GetConfig("channel-executor");
-            Priority = (TaskSchedulerPriority)Enum.Parse(typeof(TaskSchedulerPriority), cfg.GetString("priority", "normal"), true);
+            var priorityName = config.GetString("channel-executor.priority", "None") ?? "None";
```

Comment:
These changes does not have anything to do with making cluster startup to block, is it? Can you remove these and move it to a new PR?

Reply:
this is required to use the channel-executor in the spec.

I added one spec that simple starts the cluster with the channel-executor and discovered this problems.

The spec itself will not trigger the original failure, because the testkit is using a SynchronisationContext
for the akka system startup

Reply:
Yes, but this PR can easily be broken into 2 PR, one depending to the other.

Reply:
This should not be a problem anymore, fixed in #5568